ImageProcessing/src/image/geometric/Translate.md

```diff
@@ -0,0 +1,15 @@
+# image.geometric.translate
+
+The translate operator performs a geometric transformation which maps the position of each picture element in an input image into a new position in an output image. The image element located at (x1,y1) in the original is shifted to a new position (x2,y2) in the corresponding output image by displacing it through a user-specified translation (Ox,Oy).
+
```

Comment:
@Teymoor-Ali Take a look at @KirstyRD 's code:

https://github.com/robstewart57/orc-apps/blob/a11071b4812eaba351539202ba51ba9d1772cbe4/ImageProcessing/src/image/geometric/flipHorizontal.cal

It performs a horizontal flip, so the minimum data requirements to store in array buffers is a row, before it is flipped, outputted, then overwritten with the next row's values.

Though I guess your "user-specified" translation means that it's difficult to know what your memory requirements are for temporary buffers, because this is determined by that translation function I guess? Again, what are real world image processing examples of this translation user-defined'ness ?